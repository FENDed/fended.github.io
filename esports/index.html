<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <!-- Bootstrap CSS -->
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
    <!-- Font Awesome -->
    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
    <!-- Main CSS -->
    <link rel="stylesheet" href="assets/css/main.min.css">
    <title>eSports - FENDed</title>
  </head>
  <body>
    
    <header id="header" class="header">
        <div class="container">
            <div class="row">
                <div class="col-lg-6 logo">
                    <span class="name">ESPORTS ORGANISATION</span><br>
                    <span class="title">WE PLAY GAMES, PROFESSIONALLY</span>
                </div>
                <ul class="col-lg-6 nav d-flex justify-content-between">
                    <li class="nav-item"><a href="">HOME</a></li>
                    <li class="nav-item"><a href="">ROSTER</a></li>
                    <li class="nav-item"><a href="">NEWS</a></li>
                    <li class="nav-item"><a href="">STREAMS</a></li>
                    <li class="nav-item"><a href="">STORE</a></li>
                    <li class="nav-item"><a href="">CAREERS</a></li>
                </ul>
            </div>
        </div>
    </header>

    <section class="lnews-sec">
        <div class="container">
            <div class="row">
                <div class="col-12 m-auto lnews-block d-flex flex-column">
                    <span class="lnews">LATEST NEWS</span>
                    <span class="lnews-title">WELCOME TO ESPORTS ORGANISATION</span>
                    <span class="lnews-disc">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</span>
                </div>
            </div>
        </div>
    </section>

    <section class="news-sec">
        <div class="container">
            <div class="row">
                <div class="col-lg-8 news">
                    <span class="latest-news">LATEST NEWS</span>
                    <div class="news-block">
                        <img src="assets/img/news-1.png" alt="" class="news-img">

                        <div class="content d-flex">
                            <div class="news-social">
                                <div class="date d-flex justify-content-between"><i class="fa fa-clock-o"></i>22 APRIL</div>
                                <div class="views d-flex justify-content-between"><i class="fa fa-user"></i>22,462</div>
                                <div class="twitter d-flex justify-content-between"><i class="fa fa-twitter"></i>424</div>
                                <div class="like d-flex justify-content-between"><i class="fa fa-thumbs-up"></i> 771</div>
                                <a href="" class="more">READ MORE</a>
                            </div>
                            <img src="assets/img/line.png" alt="" class="news-line">
                            <div>
                                <h3 class="news-title">Call of Duty Black Ops 3 Announced!</h3>
                                <p class="news-disc">Lorem ipsum dolor sit amet, consectetur adipiscing elit,
                                    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
                                    ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
                                    ex ea commodo consequat.</p>
                            </div>
                        </div>
                    </div>

                    <div class="news-block">
                        <img src="assets/img/news-2.png" alt="" class="news-img">
    
                        <div class="content d-flex">
                            <div class="news-social">
                                <div class="date d-flex justify-content-between"><i class="fa fa-clock-o"></i>22 APRIL</div>
                                <div class="views d-flex justify-content-between"><i class="fa fa-user"></i>22,462</div>
                                <div class="twitter d-flex justify-content-between"><i class="fa fa-twitter"></i>424</div>
                                <div class="like d-flex justify-content-between"><i class="fa fa-thumbs-up"></i> 771</div>
                                <a href="" class="more">READ MORE</a>
                            </div>
                            <img src="assets/img/line.png" alt="" class="news-line">
                            <div>
                                <h3 class="news-title">ES ORG WINS HALO CHAMPIONSHIP</h3>
                                <p class="news-disc">Lorem ipsum dolor sit amet, consectetur adipiscing elit,
                                    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
                                    ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
                                    ex ea commodo consequat.</p>
                            </div>
                        </div>
                    </div>

                    <div class="news-block">
                        <img src="assets/img/news-3.png" alt="" class="news-img">
        
                        <div class="content d-flex">
                            <div class="news-social">
                                <div class="date d-flex justify-content-between"><i class="fa fa-clock-o"></i>22 APRIL</div>
                                <div class="views d-flex justify-content-between"><i class="fa fa-user"></i>22,462</div>
                                <div class="twitter d-flex justify-content-between"><i class="fa fa-twitter"></i>424</div>
                                <div class="like d-flex justify-content-between"><i class="fa fa-thumbs-up"></i> 771</div>
                                <a href="" class="more">READ MORE</a>
                            </div>
                            <img src="assets/img/line.png" alt="" class="news-line">
                            <div>
                                <h3 class="news-title">WELCOME NOOB TO THE SMITE ROSTER</h3>
                                <p class="news-disc">Lorem ipsum dolor sit amet, consectetur adipiscing elit,
                                    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
                                    ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
                                    ex ea commodo consequat.</p>
                            </div>
                        </div>
                    </div>

                    <div class="news-block">
                        <img src="assets/img/news-4.png" alt="" class="news-img">
            
                        <div class="content d-flex">
                            <div class="news-social">
                                <div class="date d-flex justify-content-between"><i class="fa fa-clock-o"></i>22 APRIL</div>
                                <div class="views d-flex justify-content-between"><i class="fa fa-user"></i>22,462</div>
                                <div class="twitter d-flex justify-content-between"><i class="fa fa-twitter"></i>424</div>
                                <div class="like d-flex justify-content-between"><i class="fa fa-thumbs-up"></i> 771</div>
                                <a href="" class="more">READ MORE</a>
                            </div>
                            <img src="assets/img/line.png" alt="" class="news-line">
                            <div>
                                <h3 class="news-title">A NEW LEAGUE OF LEGENDS TEAM RISES</h3>
                                <p class="news-disc">Lorem ipsum dolor sit amet, consectetur adipiscing elit,
                                    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
                                    ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
                                    ex ea commodo consequat.</p>
                            </div>
                        </div>
                    </div>
                </div>

                <div class="col-lg-4 aside">
                    <div class="category">
                        SPONSORS
                    </div>

                    <div class="sp sp-1 d-flex justify-content-between">
                        <img src="assets/img/sp-1.png" alt="" class="sp-logo">
                        <span class="sp-name">SCUF GAMING</span>
                    </div>
                    <div class="sp sp-2 d-flex justify-content-between">
                        <img src="assets/img/sp-2.png" alt="" class="sp-logo">
                        <span class="sp-name">ASTRO GAMING</span>
                    </div>
                    <div class="sp sp-3 d-flex justify-content-between">
                        <img src="assets/img/sp-3.png" alt="" class="sp-logo">
                        <span class="sp-name">MONSTER ENERGY</span>
                    </div>

                    <div class="category">
                        LATEST VIDEO
                    </div>

                    <img src="assets/img/video.png" alt="" class="video">

                    <div class="category">
                        DIVISIONS
                    </div>

                    <div class="div d-flex justify-content-between">
                        <img src="assets/img/d-1.png" alt="">
                        <div class="div-info d-flex flex-column">
                            <span class="div-name">CALL OF DUTY</span>
                            <a href="" class="div-link">VIEW ROSTER</a>
                        </div>
                    </div>
                    <div class="div d-flex justify-content-between">
                        <img src="assets/img/d-2.png" alt="">
                        <div class="div-info d-flex flex-column">
                            <span class="div-name">HALO</span>
                            <a href="" class="div-link">VIEW ROSTER</a>
                        </div>
                    </div>
                    <div class="div d-flex justify-content-between">
                        <img src="assets/img/d-3.png" alt="">
                        <div class="div-info d-flex flex-column">
                            <span class="div-name">SMITE</span>
                            <a href="" class="div-link">VIEW ROSTER</a>
                        </div>
                    </div>
                    <div class="div d-flex justify-content-between">
                        <img src="assets/img/d-4.png" alt="">
                        <div class="div-info d-flex flex-column">
                            <span class="div-name">LEAGUE OF LEGENDS</span>
                            <a href="" class="div-link">VIEW ROSTER</a>
                        </div>
                    </div>
                    <div class="div d-flex justify-content-between">
                        <img src="assets/img/d-5.png" alt="">
                        <div class="div-info d-flex flex-column">
                            <span class="div-name">COUNTERSTRIKE</span>
                            <a href="" class="div-link">VIEW ROSTER</a>
                        </div>
                    </div>

                    <div class="category">
                        WE RECOMMEND
                    </div>
                    <img src="assets/img/ads-1.png" alt="" class="ads">
                    <img src="assets/img/ads-2.png" alt="" class="ads">

                    <div class="category">
                        GALLERY
                    </div>
                    <div class="gallery">
                        <img src="assets/img/p-1.png" alt="" class="gal">
                        <img src="assets/img/p-2.png" alt="" class="gal">
                        <img src="assets/img/p-3.png" alt="" class="gal">
                        <img src="assets/img/p-4.png" alt="" class="gal">
                        <img src="assets/img/p-5.png" alt="" class="gal">
                        <img src="assets/img/p-6.png" alt="" class="gal">
                    </div>

                    <div class="category">
                        CURRENT STREAMS
                    </div>

                    <div class="stream">
                        <img src="assets/img/stream-1.png" alt="">
                        <div class="stream-info">
                            <span class="nick">SPUDDY</span>
                            <span class="do">PLAYING</span>
                            <span class="game">LEAGUE OF LEGENDS</span>
                        </div>

                        <img src="assets/img/stream-2.png" alt="">
                        <div class="stream-info">
                            <span class="nick">SPUDDY</span>
                            <span class="do">PLAYING</span>
                            <span class="game">COUNTERSTRIKE: GO</span>
                        </div>
                    </div>

                    <div class="category">
                        FROM TWITTER
                    </div>

                    <div class="tweet">
                        <div class="tweet-cont d-flex">
                            <img src="assets/img/tweet.png" alt="" class="tweet-img">
                            <div>
                                <span class="message">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.</span>
                                <div class="d-flex justify-content-between">
                                    <span class="time">34 MINUTES AGO</span>
                                    <span class="who">@DENIALSPUDDY</span>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>


    <!-- Optional JavaScript -->
    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
  </body>
</html>