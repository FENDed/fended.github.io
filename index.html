<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <!-- Favicon -->
    <link rel="shortcut icon" href="assets/img/favicon.ico" type="image/x-icon">
    <!-- Bootstrap CSS -->
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
    <!-- Font Awesome -->
    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
    <!-- Animate CSS -->
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.0/animate.min.css" integrity="sha256-HtCCUh9Hkh//8U1OwcbD8epVEUdBvuI8wj1KtqMhNkI=" crossorigin="anonymous" />
    <!-- Main CSS -->
    <link rel="stylesheet" href="assets/css/main.min.css">
    <!-- Fonts -->
    <link href="https://fonts.googleapis.com/css?family=Raleway:400,600,700" rel="stylesheet">
    <title>Portfolo - FENDed</title>
  </head>
  <body onload="move(); move2(); move3();">
    1
    <div id="particles-js" class="header">
      <div class="container">
        <div class="row">
          <div class="header-block" id="hblock">
            <h1 class="header-title">Hello, my nickname is <span class="color">FENDed</span></h1>
            <span class="header-text">I'm a front-end developer</span>
            <div class="btn-block m-auto">
              <a href="#about" class="header-btn ml-auto">View my portfolio<i class="fa fa-arrow-right"></i></a>
            </div>
          </div>
        </div>
      </div>
    </div>

    <section id="menu" class="menu">
      <div class="container">
        <div class="row">
          <div class="col-lg-12">
              <div class="dropdown mmenu">
                <a class="dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                  Menu
                </a>
                
                <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
                  <a class="dropdown-item" href="#particles-js">Home</a>
                  <a class="dropdown-item" href="#about">About</a>
                  <a class="dropdown-item" href="#works">My works</a>
                  <a class="dropdown-item" href="#feedback">Contact me</a>
                  <a class="dropdown-item" href="#social">Social</a>
                </div>
              </div>

            <ul class="nav col-lg-5 d-flex justify-content-between">
              <li class="nav-item"><a href="#particles-js">Home</a></li>
              <li class="nav-item"><a href="#about">About</a></li>
              <li class="nav-item"><a href="#works">My works</a></li>
              <li class="nav-item"><a href="#feedback">Contact me</a></li>
              <li class="nav-item"><a href="#social">Social</a></li>
            </ul>
          </div>
        </div>
      </div>
    </section>

    <section id="about" class="about">
      <div class="container">
        <div class="row">
          <div class="col-12">
            <h2 class="about-main wow fadeInLeft" data-wow-offset="400">About me</h2>
            <div class="about-hr m-auto wow fadeIn" data-wow-offset="400" data-wow-delay="1s" data-wow-duration="1.5s"></div>
          </div>
          <div class="col-lg-12 about-block d-flex flex-wrap">
            <div class="col-lg-6 col-12 d-flex flex-column wow fadeIn" data-wow-offset="420" >
              <span class="my-ava">My avatar</span>
              <img src="assets/img/ava.jpg" alt="" class="ava">
              <p class="ava-text">If you will see this avatar, know: it is I'm.</p>
            </div>
            <div class="col-lg-6 col-12 skills wow fadeInRight" data-wow-offset="420">
              <div id="skill-1 d-flex">
                <div id="myBar-1"><span class="skill-text">HTML</span></div>
              </div>
              <div id="skill-2 d-flex">
                <div id="myBar-2"><span class="skill-text">CSS</span></div>
              </div>
              <div id="skill-3 d-flex">
                <div id="myBar-3"><span class="skill-text">JavaScript</span></div>
              </div>
            </div>
          </div>
          <div class="col-lg-12 advantages d-flex flex-wrap">
            <div class="col-12 my-web wow fadeInLeftBig" data-wow-offset="400">
              My websites are:
            </div>
            <div class="col-lg-6 adv-item wow fadeInLeft" data-wow-offset="400" data-wow-delay="1s">
              <i class="fa fa-mobile"></i>
              <h3 class="adv-title">Responsive</h3>
              <p class="adv-text">All my websites are adaptive. You can watch them at any device: computer, laptop, tablet or phone.</p>
              <div class="line"></div>
            </div>
            <div class="col-lg-6 adv-item wow fadeInRight" data-wow-offset="400" data-wow-delay="1.5s">
              <i class="fa fa-rocket"></i>
              <h3 class="adv-title">Fast-loaded</h3>
              <p class="adv-text">All my websites are very easy to load. You can load it from low or high Internet: it is not a problem.</p>
              <div class="line"></div>
            </div>
            <div class="col-12 my-web wow fadeInLeftBig" data-wow-offset="400">
              I'm are: 
            </div>
            <div class="col-lg-6 adv-item wow fadeInLeft" data-wow-offset="350" data-wow-delay="1s">
              <i class="fa fa-book"></i>
              <h3 class="adv-title">Educable</h3>
              <p class="adv-text">I can quickly learn anything that connected with front-end development.</p>
              <div class="line"></div>
            </div>
            <div class="col-lg-6 adv-item wow fadeInRight" data-wow-offset="350" data-wow-delay="1.5s">
              <i class="fa fa-comment"></i>
              <h3 class="adv-title">Communicative</h3>
              <p class="adv-text">I can tell about all types of websites, I can help you with any problems on your website. Only write me.</p>
              <div class="line"></div>
            </div>
          </div>
        </div>
      </div>
    </section>
    
    <section class="quote wow fadeIn" data-wow-offset="500" data-wow-duration="2s">
      <div class="container">
        <div class="row">
          <q class="q">Your future is created by what you do today, not tomorrow.</q>
        </div>
      </div>
    </section>

    <section id="works" class="works">
      <div class="container">
        <div class="row">
          <div class="col-12">
            <h2 class="works-main wow fadeInLeft" data-wow-offset="400">My Works</h2>
            <div class="works-hr m-auto wow fadeIn" data-wow-offset="400" data-wow-delay="1s" data-wow-duration="1.5s"></div>
          </div>
          <div class="col-12 works-block d-flex flex-wrap">
            <a href="/golden" class="col-lg-6 work-item wow fadeInLeftBig" id="work-item-1" data-wow-offset="400"></a>
            <a href="/muncak" class="col-lg-6 work-item wow fadeInRightBig" id="work-item-2" data-wow-offset="400"></a>
            <a href="/Music fest" class="col-lg-6 work-item wow fadeInDown" id="work-item-3" data-wow-offset="400"></a>
            <a href="/shop" class="col-lg-6 work-item wow fadeInUp" id="work-item-4" data-wow-offset="400"></a>
          </div>
        </div>
      </div>
    </section>

    <section id="feedback" class="feedback">
      <div class="container">
        <div class="row">
          <div class="col-12">
            <h2 class="feedback-main wow fadeInLeft" data-wow-offset="400">Contact Me</h2>
            <div class="feedback-hr m-auto wow fadeIn" data-wow-offset="400" data-wow-delay="1s" data-wow-duration="1.5s"></div>
          </div>
          <div class="col-12">
            <h2 class="ready wow fadeInUp" data-wow-offset="400">
              Ready to contact me?
              <br>
              Just do it!
            </h2>
            <a href="https://docs.google.com/forms/d/e/1FAIpQLSeYIJ-ZRxATrW9KKHovd3C_4Jjg99sRUByN9_1b9eaQz0lgqw/alreadyresponded?c=0&amp" target="_blank"><button class="feed-btn wow shake" data-wow-offset="400" data-wow-delay="0.5s">Contact me</button></a>
          </div>
        </div>
      </div>
    </section>

    <section class="quote-2">
        <div class="container">
          <div class="row">
            <q class="q">Success doesn’t come to you… you go to it.</q>
          </div>
        </div>
      </section>

    <section id="social" class="social">
      <div class="container">
        <div class="row">
          <div class="col-12 soc-ico">
            <a href="https://vk.com/l.drel" target="_blank"><i class="fa fa-vk"></i></a>
            <a href="https://github.com/FENDed" target="_blank"><i class="fa fa-github"></i></a>
            <a href="https://codepen.io/FENDed/" target="_blank"><i class="fa fa-codepen"></i></a>
          </div>
          <div class="col-12 copy">
            Developed by <a href="https://vk.com/l.drel" class="copy-link">FENDed</a> &copy; 2018
          </div>
        </div>
      </div>
    </section>

    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js" integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm" crossorigin="anonymous"></script>
    <script src="assets/js/main.js"></script>
    <script src="assets/js/particles.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/wow/1.1.2/wow.min.js" integrity="sha256-z6FznuNG1jo9PP3/jBjL6P3tvLMtSwiVAowZPOgo56U=" crossorigin="anonymous"></script>
    <script src="assets/js/wow.js"></script>
    <script src="assets/js/progressbars.js"></script>
    <script src="assets/js/headhesive.min.js"></script>
    <script src="assets/js/stickymenu.js"></script>
    <script src="assets/js/scroll.js"></script>
    <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-114700836-4"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'UA-114700836-4');
  </script>
  </body>
</html>