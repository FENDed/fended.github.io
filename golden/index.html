<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <!-- Main CSS -->
    <link rel="stylesheet" href="assets/css/main.min.css">

    <title>Golden - FENDed</title>
</head>
<body>
    <header id="header" class="header">
        <div class="container">
            <ul class="nav">
                <img src="assets/img/logo.png" alt="" class="logo">
                <nav class="menu">
                    <li class="nav-item"><a href="#header">Home</a></li>
                    <li class="nav-item"><a href="#serv">Services</a></li>
                    <li class="nav-item"><a href="#portfolio">Portfolio</a></li>
                    <li class="nav-item"><a href="#about">About</a></li>
                    <li class="nav-item"><a href="#footer">Contact</a></li>
                </nav>
            </ul>
            <div class="htitle">
                <h2 class="htitle-little">Welcome To Our Studio!</h2>
                <h1 class="htitle-big">Itâ€™s nice to Meet you</h1>
                <button class="htitle-btn">Tell Me More</button>
            </div>
        </div>
    </header>

    <section id="serv" class="serv">
        <div class="container">
            <div class="stitle">
                <h2 class="stitle-big">Services</h2>
                <p class="stitle-text">Proin iaculis purus consequat sem cure. </p>
            </div>
            <div class="sitems">
                <div class="sitem">
                    <img src="assets/img/icon1.png" alt="">
                    <div class="sitem-block">
                        <h3 class="sitem-title">E-Ctommerce</h3>
                        <p class="sitem-text">Proin iaculis purus consequat sem cure digni ssim.
                        Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
                    </div>
                </div>
                <div class="sitem">
                    <img src="assets/img/icon2.png" alt="">
                    <h3 class="sitem-title">Responsive Web</h3>
                    <p class="sitem-text">Proin iaculis purus consequat sem cure digni ssim.
                    Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>  
                </div>
                <div class="sitem">
                    <img src="assets/img/icon3.png" alt="">
                    <h3 class="sitem-title">Web Security</h3>
                    <p class="sitem-text">Proin iaculis purus consequat sem cure digni ssim.
                    Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
                </div>
            </div>
        </div>
    </section>

    <section id="portfolio" class="portfolio">
        <div class="container">
            <div class="ptitle">
                <h2 class="ptitle-big">Our portfolio</h2>
                <p class="ptitle-text">Proin iaculis purus consequat sem cure.</p>
            </div>
            <div class="img-block">
                <img src="assets/img/img1.jpg" alt="" class="block-img">
                <img src="assets/img/img2.jpg" alt="" class="block-img">
                <img src="assets/img/img3.jpg" alt="" class="block-img">
            </div>
        </div>
    </section>

    <section id="about" class="about">
        <div class="container">
            <div class="atitle">
                <h2 class="atitle-big">About Us</h2>
                <p class="atitle-text">Proin iattculis purus consequat sem cure. </p>
            </div>
            <div class="aitems">
                <div class="aitem mr-auto">
                    <div class="aitem-block">
                        <span class="date">July 2010</span>
                        <span class="item-title">Our Humble Beginnings</span>
                        <span class="item-text ml-auto">Proin iaculis purus consequat sem cure digni ssim.
                        Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum
                        suscipit.</span>
                    </div>
                    <img src="assets/img/about1.jpg" alt="" class="aitem-img">
                </div>
                <div class="aitem ml-auto">
                    <img src="assets/img/about2.jpg" alt="" class="aitem-img">
                    <div class="aitem-block">
                        <span class="date ta-left">January 2011</span>
                        <span class="item-title ta-left">Facing Startup Battles</span>
                        <span class="item-text ta-left">Proin iaculis purus consequat sem cure digni ssim. Donec
                        porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin
                        iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt.</span>
                    </div>
                </div>
                <div class="aitem mr-auto">
                    <div class="aitem-block">
                        <span class="date">December 2012</span>
                        <span class="item-title">Enter The Dark Days</span>
                        <span class="item-text ml-auto">Proin iaculis purus consequat sem cure digni ssim. Donec
                        porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin
                        iaculis purus consequat sem cure digni.</span>
                    </div>
                    <img src="assets/img/about3.jpg" alt="" class="aitem-img">
                </div>
                <div class="aitem ml-auto">
                    <img src="assets/img/about4.jpg" alt="" class="aitem-img">
                    <div class="aitem-block">
                        <span class="date ta-left">february 2014</span>
                        <span class="item-title ta-left">Our Triumph</span>
                        <span class="item-text ta-left">Proin iaculis purus consequat sem cure digni ssim. Donec
                        porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin
                        iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
                        aenean.</span>
                    </div>
                </div>
            </div>
        </div>
    </section>

    <footer id="footer" class="footer">
        <div class="container">
            <div class="copy">&copy; 2018. Developed by <a href="https://vk.com/l.drel" target="_blank">FENDed</a></div>
        </div>
    </footer>

    <!-- Scripts -->
    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
</body>
</html>